module/motion/HeadController/src/HeadController.cpp

```diff
@@ -124,7 +124,7 @@ namespace motion {
                 //!!!!!!!!!!!!!!
                 //!!!!!!!!!!!!!!
                 std::vector<std::pair<ServoID, float>> goalAnglesList = calculateHeadJoints(headUnitVector);
-                // Eigen::Vector2f goalAngles = cartesianToSpherical(headUnitVector).block<2, 1>(1,2);
+                // Eigen::Vector2f goalAngles = cartesianToSpherical(headUnitVector).bottomRows<2>();
```

Comment:
If `cartesianToSpherical` is returning a vector then use `.tail<2>()` to get the last 2 elements
```suggestion
                // Eigen::Vector2f goalAngles = cartesianToSpherical(headUnitVector).tail<2>();
```